Assignment Day 2 _9th December 2020.ipynb

```diff
@@ -24,7 +24,7 @@
     {
      "data": {
       "text/plain": [
-       "'Mary had a little lamb'"
+       "'Mary had a little lamb and also a girl with her'"
```

Comment:
**Suggestion**: Update the text output to include both 'Mary had a little lamb' and 'a girl with her' for clarity and completeness. <br> **Code Suggestion**: 
 ```
 {
   "line": "+       \"'Mary had a little lamb and also a girl with her\"",
   "ol": null,
   "ul": 27
 }
 ```